### PR TITLE
port oidc customizations.

### PR DESCRIPTION
**Pull request by anny1099**

### CODE REVIEW

File: server/src/test/java/org/cloudfoundry/identity/uaa/login/HomeControllerViewTests.java [hunk 1]
```diff
@@ -83,6 +84,7 @@ void tearDown() {
     }
 
     @Test
+    @Disabled
```
Comment (metacubed) [inline server/src/test/java/org/cloudfoundry/identity/uaa/login/HomeControllerViewTests.java:87]: Please revert the disabled test changes - they are from the "branding" branch.

Comment (anny1099) [reply server/src/test/java/org/cloudfoundry/identity/uaa/login/HomeControllerViewTests.java]: fixed.

**Comment by irinaepshteyn** [comment]
@anny1099, I don't think we need these OIDC customizations at all.

OIDC ATTRIBUTE MAPPING (feature/oidcAttributeMapping branch) have been taken upstream since 4.6.0. PR https://github.com/cloudfoundry/uaa/pull/598

OIDC WELL KNOWN ENDPOINT(fix/oidc_well_known_endpoint branch) have been implemented in the upstream also since 4.6.0

IMPLICIT OIDC FIX (fix/implicit_oidc branch) all the commits from the branch are merged into predix_extensions_4.10.0

Am I missing anything?

cc: @metacubed, @Bitty9

**Comment by anny1099** [comment]
> @anny1099, I don't think we need these OIDC customizations at all.
> 
> OIDC ATTRIBUTE MAPPING (feature/oidcAttributeMapping branch) have been taken upstream since 4.6.0. PR [cloudfoundry#598](https://github.com/cloudfoundry/uaa/pull/598)
> 
> OIDC WELL KNOWN ENDPOINT(fix/oidc_well_known_endpoint branch) have been implemented in the upstream also since 4.6.0. Commit [cloudfoundry@4b8e9a0](https://github.com/cloudfoundry/uaa/commit/4b8e9a039d2d97b49d6268ae31c59b016bb6ef5c)
> 
> IMPLICIT OIDC FIX (fix/implicit_oidc branch) all the commits from the branch are merged into predix_extensions_4.10.0
> 
> Am I missing anything?
> 
> cc: @metacubed, @Bitty9

@irinaepshteyn , This story is for new branching model. anything has been merged into upstream is not picked, for example oidcAttributeMapping; I noticed fix/oidc_well_known_endpoint branch has additional changes not in upstream, so it is included here. When picking the changes from fix/implicit_oidc branch, also excluded the changes overlapped with brandhing.

**Comment by anny1099** [comment]
since most of changes have been covered by upstream, this can be closed.

**Pull request closed without merging**